spec/collectionspace/mapper/data_mapper_core_spec.rb

```diff
@@ -162,8 +162,9 @@
           expect(diff).to eq([]) 
         end
 
-        it 'maps as expected' do puts
```

Comment:
Don't leave any `puts` or `pp` in unless it is necessary for problem-solving. (I didn't see this outputting anything)